src/docfx/lib/HttpClientUtility.cs

```diff
@@ -11,49 +11,42 @@ namespace Microsoft.Docs.Build
 {
     internal static class HttpClientUtility
     {
-        private const int RetryCount = 3;
-        private const int RetryInterval = 1000;
         private static readonly HttpClient s_httpClient = new HttpClient();
+        private static readonly Type[] s_exceptionsToRetry = new[] { typeof(HttpRequestException), typeof(TimeoutException), typeof(OperationCanceledException) };
 
-        public static async Task<HttpResponseMessage> GetAsync(string requestUri, Config config, string etag = null)
+        public static async Task<HttpResponseMessage> GetAsync(string requestUri, Config config, EntityTagHeaderValue etag = null)
         {
-            HttpResponseMessage response;
-            for (var i = 0; i < RetryCount; i++)
-            {
-                try
+            return await RetryUtility.Retry(
```

Comment:
nit: `RetryUtility.Retry` can take a predicate instead of Type array: `Retry(Func<Task<T>> action, Func<Exception, bool> isExpectedException)`

Reply:
Is 5s retry interval too long?

Reply:
yes... mistook it in the last PR, and will change back to 1s as it is before